pkg/cluster/fetch.go

```diff
@@ -49,6 +66,18 @@ func GetBundlesForCluster(ctx context.Context, cluster *v1alpha1.Cluster, fetch
 	return bundles, nil
 }
 
+func GetFluxConfigForCluster(ctx context.Context, cluster *v1alpha1.Cluster, fetch FluxConfigFetch) (*v1alpha1.FluxConfig, error) {
+	if fetch == nil || cluster.Spec.GitOpsRef == nil {
+		return nil, nil
+	}
+	fluxConfig, err := fetch(ctx, cluster.Spec.GitOpsRef.Name, cluster.Namespace)
+	if err != nil {
+		return nil, fmt.Errorf("failed fetching FluxCOnfig for cluster: %v", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed fetching FluxConfig for cluster: %v", err)
```

Reply:
We had a recent initiative that removed words like fail and error in the error messages and make it something like `fetching FluxConfig for cluster`

Reply:
yeah I just ran out of creativity at this point :)

Reply:
fixed